libnetwork/driver_plugin.py

```diff
@@ -192,12 +358,23 @@ def join():
         remove_veth(host_interface_name)
         raise e
 
+    # TODO Optionally use the docker bridge, don't specify gateway
```

Comment:
This needs to be resolved before you can merge.

We shouldn't provide the staticroutes if the gateway is blank etc...